interface/forms/eye_mag/help.php

```diff
@@ -31,20 +31,17 @@
 ?>
 <html>
     <head>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta name="description" content="Eye Exam Help">
-    <meta name="author" content="openEMR: ophthalmology help">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="Eye Exam Help" />
+    <meta name="author" content="openEMR: ophthalmology help" />
     <?php Header::setupHeader(['jquery-ui', 'jquery-ui-excite-bike']); ?>
 
-    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/pure/0.5.0/pure-min.css">
+    <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative'] ?>/purecss/pure-min.css">
```

Comment:
Testing: Pretty sure pure css is gone lol

Reply:
still there, but that is pointed to wrong path. We should remove this and place 'purecss' in the above Header:setupHeader call.